Cargo.toml

```diff
@@ -441,6 +441,7 @@ aptos-cargo-cli = { path = "devtools/aptos-cargo-cli" }
 
 # External crate dependencies.
 # Please do not add any test features here: they should be declared by the individual crate.
+aes-gcm = "0.10.3"
```

Comment:
Why not use `ring` which we already depend on?

Reply:
added a TODO to take a deeper look at both later.